fs/treewalk.go

```diff
@@ -1241,7 +1241,7 @@ func (sVS *scrubVolumeStruct) scrubVolumeInode(inodeNumber uint64) {
 
 	defer sVS.jobReleaseParallelism()
 
-	inodeLock, err = sVS.volume.initInodeLock(inode.InodeNumber(inodeNumber), nil)
+	inodeLock, err = inode.InitInodeLock(sVS.jobStruct.volumeName, inode.InodeNumber(inodeNumber), nil)
```

Comment:
I guess this surprised me... I thought you had said you moved this API to DLM... and not INODE.

As for adding APIs to package Inode, I'm quite a bit more critical... as all of the existing APIs in package Inode are methodically tested... so I'll have a high bar for agreeing that an API needs to be added. Let's see the test code for this...

Reply:
Fix per comments below...